bodhi/server/buildsys.py

```diff
@@ -340,7 +340,7 @@ def listTags(self, build, *args, **kw):
                  'perm': None, 'perm_id': None},
                 {'arches': 'i386 x86_64 ppc ppc64', 'id': 5, 'locked': True,
                  'name': '%s-testing' % release, 'perm': None, 'perm_id': None}]
-        elif '-master-' in build:
+        elif '-master-' in build or build.startswith(('nodejs-6-', 'nodejs-8-', 'nodejs-9-')):
```

Comment:
This seems oddly hardcoded, is it expected?

Reply:
Yes, that's how the whole test buildsys determines whether things are a module etc.
This is just the DevBuildsystem.

Reply:
Yeah, the DevBuildsystem is full of oddly hardcoded things.  :+1: